chainercv/links/detection_link.py

```diff
@@ -0,0 +1,39 @@
+import chainer
+
+
+class DetectionLink(chainer.Link):
```

Comment:
Since there is no actual code in `DetectionLink`, we may not need it.
It was still fruitfull trying to implement it and deepen our understandings on API of the predict function.